### PR TITLE
chore: visual refresh on breadcrumb item component

### PR DESCRIPTION
**Pull request by khamudom**
# Pull Request

## 📖 Description

Removed the accent color, update the visual states, and the spacing between items.

before (rest, hover active, focus)
image

after
image

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

File: packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts [hunk 1]
```diff
@@ -33,113 +36,125 @@ export const breadcrumbItemStyles: (
...
+                color: ${SystemColors.LinkText};
+                fill: ${SystemColors.ButtonText};
+                border-color: ${SystemColors.LinkText};
+              }
+              .control .content::after {
+                content: "";
+                display: block;
+                height: calc(${strokeWidth} * 1px);
+              }
+              .control:hover .content::after ,
```
Comment (janechu) [inline packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts:147]:
```suggestion
              .control:hover .content::after,
```

File: packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts [hunk 2]
```diff
@@ -33,113 +36,125 @@ export const breadcrumbItemStyles: (
...
+      border-radius: inherit;
     }
-
     .separator {
-        margin: 0 6px;
-        display: flex;
-    }
-
+        margin: 0 20px;
+        display:flex;
```
Comment (janechu) [inline packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts:57]:
```suggestion
        display: flex;
```

File: packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts (hunk 1 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts]: updated

File: packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts (hunk 2 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts]: updated

File: packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts [hunk 3]
```diff
@@ -33,113 +36,125 @@ export const breadcrumbItemStyles: (
...
-        display: flex;
-        fill: inherit;
-        outline: none;
-        text-decoration: none;
-        white-space: nowrap;
+      position: relative;
+      align-items: center;
+      box-sizing: border-box;
+      background: transparent;
+      border: calc(${strokeWidth} * 1px) solid transparent;
```
Comment (bheston) [inline packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts:64]:
```suggestion
      border: calc(${focusStrokeWidth} * 1px) solid transparent;
```

File: packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts [hunk 4]
```diff
@@ -33,113 +36,125 @@ export const breadcrumbItemStyles: (
...
     }
-
     .control:active {
-        color: ${accentForegroundActive};
+      background: ${neutralFillStealthActive};
+      color: ${neutralForegroundActive};
+    }
+    .control:${focusVisible} {
+      border-color: ${focusStrokeOuter};
+      box-shadow: 0 0 0 calc(${focusStrokeWidth} * 1px) ${focusStrokeOuter} inset;
```
Comment (bheston) [inline packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts:84]:
```suggestion
```

File: packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts [hunk 5]
```diff
@@ -33,113 +36,125 @@ export const breadcrumbItemStyles: (
...
     .control .content::before {
-        content: "";
-        display: block;
-        height: calc(${strokeWidth} * 1px);
-        left: 0;
-        position: absolute;
-        right: 0;
-        top: calc(1em + 4px);
-        width: 100%;
+      content: "";
```
Comment (bheston) [inline packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts:91]: Is this the link underline? Can we use text-decoration instead?

Comment (khamudom) [reply packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts]: switched to use text decoration

File: packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts (hunk 4 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts]: removed

File: packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts (hunk 3 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts]: updated

**Review by bheston — APPROVED** [review]

**Review by scomea — APPROVED** [review]

**Pull request merged**